services-js/commissions-app/src/lib/validationSchema.ts

```diff
@@ -1,17 +1,14 @@
-import * as Yup from 'yup';
+// polyfill Array.from() for IE
+import 'core-js/fn/array/from';
```

Comment:
I think it's worth mentioning that the polyfill is due to Yup needing it.

Reply:
You are so right.